### DISCUSSION TITLE
Dynamic properties in LiveWire

### DISCUSSION THREAD

**Discussion by JunaidQadirB** [livewire/livewire#2767] [1. Help]
I'm working on a component that will get properties based on the data passed to it.
Right now if I pass that data Livewire throws the `Livewire\Exceptions\PropertyNotFoundException`.
This suggests that I cannot declare props in runtime. 

So if I have to create dynamic properties what would be the best way?
In case that's not possible, what route should I take?

Thanks in advance🙏

**Comment by Bakusedu** [comment]
A detailed snapshot of your code indicating what you're encountering would help, also make sure that the props you are passing is being declared as a `public $property` in the LivewireComponent you are passing the props to

**Reply by JunaidQadirB** [reply]
Thank you for your reply. That's what I don't want, to declare the props beforehand, hence using the term 'Dynamic properties'.
I'll try to to share the code as I get time later in the day.
